public/static/docs/command-reference/run.md

```diff
@@ -79,10 +79,10 @@ data pipeline (e.g. random numbers, time functions, hardware dependency, etc.)
 
 ## Options
 
-- `-d`, `--deps` - specify a file or a directory the stage depends on. Multiple
-  dependencies can be specified like this: `-d data.csv -d process.py`. Usually,
-  each dependency is a file or a directory with data, or a code file, or a
-  configuration file. DVC also supports certain
+- `-d <path>`, `--deps <path>` - specify a file or a directory the stage depends
```

Comment:
just to check - can it be a comma-separated list?

Reply:
Not for `--deps`, but it can be in other options. That is shown in the Synopsis though, not here. I think the only one is `dvc gc -p.`

Reply:
Please merge this if you're satisfied with the answer 🙂

Reply:
yeah, that's right  ... it's more about options that could be specified once vs those that could be specified multiple time ... not sure if we need a special syntax to highlight that

Reply:
Ah good question. I didn't even think about that but it seems like a relevant thing to note somewhere. Opened #1041